microshift_configuring/microshift_low_latency/microshift-low-latency.adoc

```diff
@@ -0,0 +1,77 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-low-latency"]
+= Configuring low latency
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-low-latency
+
+toc::[]
+
+You can configure and tune low latency capabilities to improve application performance on edge devices.
+
+include::modules/microshift-low-latency-concept.adoc[leveloffset=+1]
+
+//additional resources for the low latency concept
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/scalability_and_performance/low-latency-tuning#cnf-understanding-low-latency_cnf-understanding-low-latency[About low latency] (OpenShift Container Platform documentation)
+
+include::modules/microshift-install-rpms-low-latency.adoc[leveloffset=+1]
+
+include::modules/microshift-low-latency-config-yaml.adoc[leveloffset=+1]
+
+//additional resources for the config.yaml
+[role="_additional-resources"]
+.Additional resources
+* xref:../../microshift_configuring/microshift-using-config-tools.adoc#microshift-config-yaml_microshift-configuring[Using a YAML configuration file]
+* link:https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration[KubeletConfiguration reference] (Kubernetes upstream documentation)
+* link:https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#create-the-config-file[Create the config file] (Kubernetes upstream documentation)
```

Comment:
I'm wondering if this isn't too confusing as it mentions `apiVersion` and `kind` but we don't want those.

Reply:
Removed